lib/ruby_smb/client.rb

```diff
@@ -332,7 +332,7 @@ def initialize(dispatcher, smb1: true, smb2: true, smb3: true, username:, passwo
       # session setup response is received
       @session_encrypt_data = always_encrypt
 
-      @ntlm_client = Net::NTLM::Client.new(
+      @ntlm_client = NTLM::Client.new(
```

Comment:
```suggestion
      @ntlm_client = RubySMB::NTLM::Client.new(
```

Just for explicitness, as I think this might trip up future code-readers if they're not aware of the pattern being used here 😄